docs/real-world-examples/ner.md

```diff
@@ -0,0 +1,53 @@
+# Named Entity Recognition using MultiNERD dataset
+
+## Background
+This example shows how to train a federated model for the Named Entity Recognition task. This tutorial uses the [MutliNERD](https://github.com/Babelscape/multinerd/blob/master/README.md) dataset.  
+The steps listed below describe a step-by-step process for carrying out this experiment, including provisioning resources, uploading data, creating training scripts, etc. Any NLP work can be carried out in a federated fashion using this example as a template.
+
+> For the sake of simplicity, we will only provision an _open_ setup. Do not upload sensitive data to it! 
+## Prerequisites
+To enjoy this tutorial, you will need to:
+- have an active [Azure subscription](https://azure.microsoft.com) that you can use for development purposes,
+- have permissions to create resources, set permissions, and create identities in this subscription (or at least in one resource group),
+  - Note that to set permissions, you typically need _Owner_ role in the subscription or resource group - _Contributor_ role is not enough. This is key for being able to _secure_ the setup.
+- [install the Azure CLI](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli).
+
+## Procedure
+Please follow the below instructions to provision resources, upload data, and then run the experiment:
+
+### Provision Resources
+The instructions are provided in the [quickstart](../quickstart.md) to provision an open sandbox. Make note of the name of the resource group you provisioned, as well as the name of the workspace.
+
+### Upload Data
```

Comment:
One thing that is the same across all three examples is that, the data-upload section is only applicable for "running the examples" and not expected applicable for real FL scenario, and they are provided here simply for the only purpose of convenience of running the FL example for the customer, and users should ignore this section when they want to do their real FL experiment for their scenario. (In real FL scenario, the data from all silos would never be in a single central location, and the 'data-uploading' section in all the three examples are not expected to be applicable here; instead, each silo owner are expected to upload there own private data to their own private & secure datastores.) It might be helpful to explicitly point this out somewhere, maybe under docs/README.md, to avoid any potential confusions for the 3P customers.

Reply:
Added a note in the Readme